### ISSUE TITLE
indicator of indicator - Heikin-Ashi pattern

### ISSUE THREAD

**Issue by moslem7026** [DaveSkender/Stock.Indicators#717]
when candle stick pattern convert to Heikin-Ashi , and the want to get indictor on Heikin-Ashi pattern . it is not possible.
 `var i = Candlse.GetHeikinAshi().ToList();` it return `List<HeikinAshiResult>` but Indicator.GetStochRsi() needs `List<Qoute>`

**Comment by DaveSkender** [comment]
Do it like this:

```csharp
var results = quotes
          .GetHeikinAshi()
          .ConvertToQuotes()
          .GetStochRsi();
```

**Comment by DaveSkender** [comment]
@moslem7026 you've inspired me to take another look at this.  I'm going to base both `HeikinAshiResult` and `RenkoResult` on `IQuote` in the library.  Once implemented, you'll be able to skip the conversion step and simply use the results directly in other indicators; example:

```csharp
var haQuotes = quotes
          .GetHeikinAshi();

var haRsi = haQuotes.
          .GetRsi();
```

**Comment by DaveSkender** [comment]
These updates are now available in v1.21.1
